lib/utils.js

```diff
@@ -0,0 +1,36 @@
+const isPlainObj = require('is-plain-obj');
+
+const wrapArg = item => {
+	const alfredworkflow = {arg: item.arg, variables: item.env};
+	const arg = JSON.stringify({alfredworkflow});
+	const newItem = Object.assign({}, item, {arg});
+	delete newItem.env;
```

Comment:
Can we overwrite `env` with `undefined` instead in the line above? If I'm not mistaken `delete` is bad for performance.